packages/components/src/query-controls/tag-select.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import { buildTermsTree } from './terms';
+import TreeSelect from '../tree-select';
+
+export default function TagSelect( { label, noOptionLabel, tagsList, selectedTagId, onChange } ) {
+	const termsTree = buildTermsTree( tagsList );
+	return (
+		<TreeSelect
```

Comment:
I wonder if this is the best form input to use. Isn't the one used at document level tag assignment better `TokenInput` ?